src/sql/workbench/contrib/notebook/browser/notebookStyles.ts

```diff
@@ -53,6 +52,11 @@ export function registerNotebookThemes(overrideEditorThemeSetting: boolean, conf
 				// Override values only for the children of code-component so regular editors aren't affected
 				collector.addRule(`code-component .monaco-editor .view-overlays .current-line { border: 0px; }`);
 			}
+		} else {
+			let lineHighlight = theme.getColor(cellEditorLineHighlight);
+			if (lineHighlight) {
+				collector.addRule(`code-component .monaco-editor .view-overlays .current-line { background-color: ${lineHighlight}; border: 0px; }`);
```

Comment:
Can the `border: 0px` be pulled from here and added to an existing class in the CSS?